.github/scripts/update_releases.py

```diff
@@ -14,7 +14,7 @@
     print_file_summary,
     print_progress_bar,
     write_file,
-    get_json_from_github,
+    get_json_from_github, add_file_group,
```

Comment:
Minor detail but this should go on a new line

Reply:
Good spot... PyCharm nicely adds new imports, but doesn't honour the existing formatting, and I missed that one.

Now fixed.